pom.xml

```diff
@@ -341,6 +341,11 @@
                 <artifactId>guice-multibindings</artifactId>
                 <version>${guice.version}</version>
             </dependency>
+            <dependency>
```

Comment:
What is this? Why this is needed?

Reply:
Future versions of guava fail to pull this in. 19 specifically if I recall. This causes builds to fail for guava 19 without it.